### PR TITLE
Public visibility for URLJSONFormatter and JSONKeypath resolve.

### PR DESCRIPTION
**Pull request by mpurland**

### CODE REVIEW

File: Tyro/JSONKeypath.swift [hunk 1]
```diff
@@ -51,7 +51,7 @@ extension JSONKeypath : CustomStringConvertible {
 }
 
 extension JSONKeypath {
-    func resolve(dictionary : Dictionary<String, JSONValue>) -> JSONValue? {
+    public func resolve(dictionary : Dictionary<String, JSONValue>) -> JSONValue? {
```
Comment (CodaFi) [inline Tyro/JSONKeypath.swift:54]: This should definitely be rewritten without recursion (sorry!)

**Comment by mpurland** [comment]
👍 ⛵

File: Tyro/JSONKeypath.swift (hunk 1 above)
Comment (mpurland) [inline Tyro/JSONKeypath.swift:54]: Definitely. I thought it would be good to made public so that it can be used in outside implementations if necessary.

**Comment by CodaFi** [comment]
We ought to protect master and work around that weird build failure.  It's freaking me out.

**Comment by mpurland** [comment]
Same here. Need to figure that one out.

**Pull request merged**